server/src/models/Match.ts

```diff
@@ -0,0 +1,76 @@
+import { getSettings } from '@src/services/SettingsService'
+
+export class Match {
+  constructor (
+    public matchKey: string,
+    public eventKey: string,
+    public compLevel: 'qm' | 'sf' | 'f',
+    public setNumber: number,
+    public matchNumber: number,
+
+    public startTime: EpochTimeStamp,
+    public endTime: EpochTimeStamp,
+
+    public blue_keys: string[],
+    public red_keys: string[],
+
+    public blueScore: number,
+    public redScore: number,
+
+    public winner: 'blue' | 'red' | ''
+  ) { }
+
+  private static readonly compLevelString = {
+    qm: 'Qualification Match',
+    // qf: 'Quarterfinals Match',
+    // sf: 'Playoffs Match',
+    f: 'Finals Match'
+  }
```

Comment:
No need to resolve now, but we may well need to revisit this structure because blindly calling `sf` match codes Playoffs may not even scale if TBA changes their data format next year